frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPaletteSearchInput/CommandPaletteSearchInput.module.css

```diff
@@ -31,20 +31,34 @@
   flex-shrink: 0;
   margin-right: var(--spacing-1);
   padding: var(--spacing-2) 0;
+  max-width: 220px;
```

Comment:
I confirmed that in most cases, table names with three words or fewer should be rendered entirely.

<img width="731" height="151" alt="Image" src="https://github.com/user-attachments/assets/57f33e85-8785-4767-aa2b-9609eefc9b10" />